src/fragments/forms/settings/i18n/settings.i18n.cs-cz.js

```diff
@@ -0,0 +1,50 @@
+
+export default {
+  settings: {
+    endpointsSectionTitle: 'Koncové body služby (URI nebo úplná adresa URL)',
+    language: 'Jazyk',
+    appLocale: 'Lokalizace aplikace',
+    unit: 'Jednotka',
+    areaUnit: 'Jednotka plochy',
+    saveToLocalStorage: 'Uložit v mém prohlížeči',
+    routeInstructions: 'Instrukce pro navigaci',
+    randomizedIsochroneColors: 'Náhodné izochronní barvy',
+    distanceMarkers: 'Značky vzdálenosti',
+    elevationProfile: 'Profil převýšení',
+    mapSettings: 'Zobrazení mapy a další informace',
+    defaultMapSettingsRestored: 'Obnovení a uložení výchozího nastavení mapy',
+    endpointsTitle: {
+      directions: 'Instrukce',
+      isochrones: 'Isochrony',
+      geocodeSearch: 'Hledání podle Geocode',
+      autocomplete: 'Automatické dokončení Geocode',
+      pois: 'Body zájmu',
+      reverseGeocode: 'Reverzní Geocode'
+    },
+    apiKey: 'klíč API',
+    customApiKey: 'Vlastní klíč API',
+    apiBaseUrl: 'Základní URL API',
+    invalidSettingsValue: 'Neplatné hodnoty nastavení. Zkontrolujte pole nebo obnovte výchozí hodnoty',
```

Comment:
The original message reads "value(s)", suggesting that it may be one or more faulty values. Automatic translation suggests that this here uses the plural. Is there an option to denote that it may be one or more values in czech?